### ISSUE TITLE
[UX Enhancement] Disable Gravity/Physics

### ISSUE THREAD

**Issue by flowpoly** [mozilla/hubs#1489]
As a user, I can enable gravity/physics from the object's menu, but how do I disable it? 
![Physics](https://user-images.githubusercontent.com/38146102/60178229-fcbd9300-97e8-11e9-9a1d-fc9db5b8c8b9.jpg)

**Comment by gfodor** [comment]
The main reason I didn't implement the inverse operation is because the button typically will be under the floor once the object has been gravity-enabled, and you can't pick it up to click on it either since gravity is always on, so it seemed like it could be very frustrating (vs just teaching users its invertible.) there might be a better design tho.

**Comment by flowpoly** [comment]
@gfodor Oh! So if I am using VR, I can't pick it up due to gravity?

**Comment by gfodor** [comment]
You can pick it up, but it'll drop as soon as you let go, so there is no way to easily hit the button.

**Comment by flowpoly** [comment]
If the object's menu was always looking at the user and it was triggered by pressing a button in VR just like in PC (spacebar), could it be possible to add a button that represents 'disable gravity'?

**Comment by misslivirose** [comment]
I agree that the setting should be able to be reverted - and that changing the menu orientation on objects so that it's not blocked by the ground surface makes sense, so this suggestion is dependent on first addressing #1460.

**Comment by gfodor** [comment]
yup that makes sense. to be clear when I build this feature the button on the bottom was originally a toggle, and I found it unusable, so in the interest of shipping the "turn on gravity" function I changed it to remove the button as a temporary solution. I agree we should fix the menus somehow so there's no proximity-to-ground dependency on UX. (though I believe the main idea proposed for #1460 may not address that properly -- since doing a cylindrical billboarding will still end up clipping into the floor unless we offset the menu also based upon environmental factors in addition to object and user camera state.)

**Comment by gfodor** [comment]
the other approach would be to not avoid the clipping per se but turn the environment transparent or just draw it anyway, but you could run into weird convergence issues or other UX problems -- also perf issues with that. hard to know how bad it would be unless we tried it.
